exercises/practice/state-of-tic-tac-toe/state_of_tic_tac_toe_test.go

```diff
@@ -0,0 +1,27 @@
+package stateoftictactoe
+
+import (
+	"testing"
+)
+
+func TestStateOfTicTacToe(t *testing.T) {
+	for _, c := range testCases {
+		t.Run(c.description, func(t *testing.T) {
+			result, err := StateOfTicTacToe(c.board)
+			if c.wantErr {
```

Comment:
This if statement handles the case where we want an error but we didn't get one.

We must also test and throw an appropriate error message for when there wasn't an error returned by the student but we expected one.